src/lib/components/workflow/batch-operation-confirmation-modal.svelte

```diff
@@ -44,7 +44,7 @@
 </script>
 
 <Modal
-  id="batch-operation-confirmation-modal"
+  id={`batch-operation-confirmation-modal-${action}`}
```

Comment:
Small thing, but we can make these a tiny bit simpler like so

```suggestion
  id="batch-operation-confirmation-modal-{action}"
```